src/types/types.ts

```diff
@@ -1311,8 +1315,9 @@ export interface EdgeSwapPlugin {
   readonly fetchSwapQuote: (
     request: EdgeSwapRequest,
     userSettings: JsonObject | undefined,
-    opts: { promoCode?: string }
+    opts: { infoPayload: JsonObject; promoCode?: string }
   ) => Promise<EdgeSwapQuote>
+  readonly updateInfoPayload?: (infoPayload: JsonObject) => Promise<void>
```

Comment:
I thought were weren't going to have the updateInfoPayload method on the swap plugins?